### DISCUSSION TITLE
Any plans for Websockets support?

### DISCUSSION THREAD

**Discussion by Maxzilla60** [layrjs/layr#8] [Ideas]
First of all, love Layr! Looking forward to what the next version has in store! 😁

Pretty much what the title says. Not sure if it would even be feasible?

**Comment by mvila** [comment]
Thanks, @Maxzilla60! Real-time communication between the frontend and the backend via WebSockets is on the roadmap indeed, but it's not a small deal, and I cannot say when it will be implemented.
